src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -11536,7 +11508,11 @@ protected void UpdateBounds(int x, int y, int width, int height, int clientWidth
 #if DEBUG
                 if (Bounds != originalBounds && CompModSwitches.SetBounds.TraceWarning)
                 {
-                    Debug.WriteLine(string.Format(CultureInfo.CurrentCulture, "WARNING: Bounds changed during OnLocationChanged()\r\nbefore={0} after={1}", originalBounds, Bounds));
+                    Debug.WriteLine($"""
+                        WARNING: Bounds changed during OnLocationChanged()
+
+                        before={originalBounds} after={Bounds}
```

Comment:
There was only one line break previously, the empty line is redundant:
```suggestion
                        WARNING: Bounds changed during OnLocationChanged()
                        before={originalBounds} after={Bounds}
```

Reply:
You're right, thanks. Fixed line 11537 as well